package.json

```diff
@@ -44,13 +45,14 @@
   ],
   "license": "MIT",
   "main": "ampersand-sync.js",
+  "browser": "ampersand-sync-browser.js",
```

Comment:
I don't believe that format works with webpack by default, that was my main concern with this approach.